app-admin/sudo/README.md

```diff
@@ -0,0 +1,8 @@
+## Flatcar changes
+- Remove Perl Runtime Dependency
+- Remove OpenLDAP schema files for sudo
+```
+insinto /etc/openldap/schema
+newins doc/schema.OpenLDAP sudo.schema
+```
```

Comment:
If you comment these lines out instead of removing, then you won't need to add them here.